### PR TITLE
.swiftlint.yml: cleaned old folders

### PR DESCRIPTION
**Pull request by NachoSoto**
Follow up to #1335.

### CODE REVIEW

File: .swiftlint.yml [hunk 1]
```diff
@@ -1,11 +1,7 @@
 excluded:
   - Carthage
-  - Pods
   - Examples
   - IntegrationTests
-  - PublicSDKAPITester
-  - vendor
-  - scan_derived_data
```
Comment (NachoSoto) [inline .swiftlint.yml:8]: I had this folder locally but it's no longer in git. Is it created by something I'm not aware of?

Comment (aboedo) [reply .swiftlint.yml:8]: we had it created in a lane in fastlane, don't know whether it's still around or not

Comment (aboedo) [reply .swiftlint.yml:8]: yep, `backend_integration_tests`

Comment (NachoSoto) [reply .swiftlint.yml:8]: Okay I'll leave this one then.

**Review by aboedo — APPROVED** [review]

**Pull request merged**